scripts/release-bit-verification.sh

```diff
@@ -11,16 +11,32 @@
 #
 # Example: ./release-bit-verification.sh ~/Downloads/odo-redistributable-2.4.3-1.el8.x86_64.rpm
 #
+#For erroring out in case of error
+set -eo pipefail
 
 shout() {
     echo "--------------------------------$1------------------------------------------"
 }
 # Check SHASUM for all the binary files and there should be no difference
 
+
+
 # Create a Temp directory
+if [ "$#" -lt 1 ]
+then
+  echo "No arguments supplied"
+  exit 1
+fi
+
 WORKING_DIR=$(mktemp -d)
-shout "WORKING_DIR=$WORKING_DIR"
-export REPO_URL=${REPO_URL:-"https://github.com/redhat-developer/odo.git"}
+if [ -f ${1} ]; 
```

Comment:
This would still create the temporary directory if the file is not valid.
I'd suggest inverting the condition: exit if the file does not exist, without even creating the temporary directory.

Once all checks are done on the first argument, we can proceed with creating the temporary directory.